src/main/java/com/example/online_shop_api/Service/PasswordService.java

```diff
@@ -0,0 +1,72 @@
+package com.example.online_shop_api.Service;
+
+import com.example.online_shop_api.Entity.Employee;
+import com.example.online_shop_api.Entity.User;
+import com.example.online_shop_api.Exceptions.IncorrectPasswordException;
+import com.example.online_shop_api.MyUserDetails;
+import com.example.online_shop_api.Repository.EmployeeRepository;
+import com.example.online_shop_api.Repository.UserRepository;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.stereotype.Service;
+
+@Service
+public class PasswordService {
+
+    private EmployeeRepository employeeRepository;
```

Comment:
Is it intentional that we don't have any kind of dependency injection for the 3 private properties in this class?

Reply:
I added the @AllArgsConstructor annotation after my last commit. I will push the latest version with it.